app/forms/add_to_order_form.rb

```diff
@@ -230,15 +230,33 @@ def merge_order
                        created_by: created_by.id,
                        cross_core_project: @order_project,
                      )
-                   else
+                    else
                      target_order
-                   end
+                    end
+  end
+
+  def find_or_create_product_order
+    return order_for_selected_facility if order_for_selected_facility.present?
+
+    Order.create!(
+      facility: product_facility,
+      account_id:,
+      user_id: original_order.user_id,
+      created_by: created_by.id,
+      cross_core_project: @order_project
+    )
+  end
+
+  def requires_merge?
```

Comment:
Comment - intention is to determine if we should create a merge order or not based on the products